content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/running.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Running Open 3D Multiplayer Projects
+description: How to run Multiplayer enabled projects and relevant configuration
+linkTitle: Running Multiplayer
+---
+
+Running projects using the Open 3D Engine Multiplayer Gem requires build steps followed by a variety of options to run the projects.
+
+* Ensure your project is [properly configured](./configuration.md)
+* Build your project's `GameLauncher` and `ServerLauncher`
+* Use one of the following methods to run
+
+## In Editor Testing
+
+Multiplayer projects can be [run in Editor](./test-in-editor.md) using Ctrl-G. See the accompanying documentation for more detail. 
+
+## Running locally using ImGui Options
+
+1. Launch your `ServerLauncher` either from Visual Studios or from your build directory.
+2. Press the `HOME` key to enable the debug menu.
+3. Select the `Host` option.
+4. Select a level to load to begin hosting.
+5. Use the `Launch Local Client` option to automatically launch a client connected to the server.
+
+## Running Server and Client Launchers and configuring via console 
```

Comment:
Maybe split into?

## Running Server and Client launcher in Standalone

### Manual configuration using the application's console

### Using pre-defined config files